site/src/components/ExpandableMenuItem/ExpandableMenuItem.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { ChevronRight } from '@sparkpost/matchbox-icons';
+import { Box } from '@sparkpost/matchbox';
+import { navigate } from 'gatsby';
+import { tokens } from '@sparkpost/design-tokens';
+
+const StyledHeader = styled(Box)`
+  cursor: pointer;
+`;
+
+const StyledChevronRight = styled(ChevronRight)`
+  transform: ${props => (props.expanded ? 'rotate(90deg)' : 'rotate(0deg)')};
+  transition: transform ${tokens.motionDuration_fast}
+    ${tokens.motionEase_in_out};
+`;
+
+function ExpandableMenuItem(props) {
+  const { value, defaultExpanded, firstRoute, children } = props;
+
+  const [expanded, setExpanded] = useState(defaultExpanded || false);
+
+  function toggleExpanded() {
+    setExpanded(!expanded);
+  }
+
+  function navigateToFirstLink() {
+    if (!defaultExpanded) {
+      navigate(firstRoute);
```

Comment:
This is good, but feels a little janky - i found this, I think it would be good to incorporate:
https://www.gatsbyjs.org/docs/browser-apis/#shouldUpdateScroll

in gatsby browser file:
```js
exports.shouldUpdateScroll = ({
  routerProps: { location },
  getSavedScrollPosition
}) => {

 // const currentPosition = getSavedScrollPosition(location);
 // Maybe we can do a smooth scroll to top here on route change?
  return false;
};
```

Reply:
I was thinking of implementing something to behave similar to the polaris site – it does a smooth scroll to the top on route change.  we'd need to test with #` anchor links too to make sure those still work